CHANGES/5227.misc

```diff
@@ -0,0 +1 @@
+Removing latest filter Collection Version.
```

Comment:
I think this needs to be named `5227.removal` so it's listed in the breaking changes section of the release notes.

Reply:
Also maybe put latest in single quotes, double quotes, or backticks.